docs/docs/admin/teamforcapella/migration.md

```diff
@@ -3,21 +3,26 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-# How to Update a Capella Model to a Higher Version (in TeamForCapella)
+# How to Migrate a Capella Model to a Higher Version (in TeamForCapella)
 
 !!! warning
 
-    Please create backups (e.g., in a Git repository) before you start the update process.
     A downgrade of Capella models is not possible.
 
 1.  Open a new persistent session with the old Capella version.
+1.  Check that nobody is connected to the remote repository:
+    [See who is connected to a TeamForCapella repository](../../user/tools/capella/teamforcapella/connect/index.md)
+1.  Run a backup pipeline and verify that the latest changes are included in
+    the Git repository.
 1.  Import the model from the TeamForCapella server via the Import wizard from
     TeamForCapella into your local workspace:
     [Import a model from TeamForCapella](../../user/tools/capella/teamforcapella/import/index.md)
 1.  Terminate the session.
-1.  Remove existing backups from TeamForCapella:
+1.  Remove existing TeamForCapella to Git backup pipelines in the Collaboration
+    Manager and note down if the commit history is included:
```

Comment:
Why is this needed, when we restore later anyways? Is it imaginable to automate an update of existing backup pipeline records somehow!?

Reply:
A backup pipeline always has a fixed source (TeamForCapella repository) and a target (Git repository). A repository can't be deleted if a backup pipeline still references it.

Reply:
Issue: https://github.com/DSD-DBS/capella-collab-manager/issues/1749